src/main/java/redis/clients/jedis/commands/AdvancedBinaryJedisCommands.java

```diff
@@ -64,5 +64,11 @@
 
   List<byte[]> aclCat(byte[] category);
 
+  List<byte[]> aclLogBinary();
```

Comment:
`List<byte[]>` doesn't seem right. Take a look at `hgetAll(byte[])`. If nothing works, simple `Object` may be better.

Reply:
The reason why I have create a list is because of the way the log is organized: (and I can if you think it is better create an object like we do for StreamEntry - but his list of "map" is quite generic and simple)
```
127.0.0.1:6379> acl log
1)  1) "count"
    2) (integer) 1
    3) "reason"
    4) "key"
    5) "context"
    6) "toplevel"
    7) "object"
    8) "foo-2"
    9) "username"
   10) "antirez"
   11) "age-seconds"
   12) "25.146999999999998"
   13) "client-info"
   14) "id=34 addr=127.0.0.1:53509 fd=11 name= age=0 idle=0 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=33 qbuf-free=32735 obl=0 oll=0 omem=0 events=r cmd=set user=antirez"
2)  1) "count"
    2) (integer) 5
    3) "reason"
    4) "command"
    5) "context"
    6) "toplevel"
    7) "object"
    8) "incr"
    9) "username"
   10) "antirez"
   11) "age-seconds"
   12) "25.146999999999998"
   13) "client-info"
   14) "id=34 addr=127.0.0.1:53509 fd=11 name= age=0 idle=0 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=23 qbuf-free=32745 obl=0 oll=0 omem=0 events=r cmd=incr user=antirez"
```

Reply:
Oh yes... I oversimplified this part and I will add the proper processing and more tests about binary

Reply:
I was looking at `BinaryJedis.xrange()`, that returns also a list of `byte[]`.

However,  I will create a new class `AccessControlLogEntry` that will be cleaner overall, and I will use it also in this binary class. (for example `BinaryJedis.xinfoGroup` returns List<StreamGroupInfo>

I think the ACL LOG is more useful was a specific class even in binary class

Reply:
*Considering you want to stick to this,* `List<byte[]>` isn't proper return type. It has to be at least `List<Object>`. So just return `Object`*.
\* You may have to consider Connection.getOne().